compiler/control/OMROptions.cpp

```diff
@@ -1075,6 +1075,7 @@ TR::OptionTable OMR::Options::_jitOptions[] = {
                                          TR::Options::setStaticNumericKBAdjusted, (intptr_t)&OMR::Options::_scratchSpaceLowerBound, 0, "F%d (bytes)"},
    {"searchCount=",      "O<nnn>\tcount of the max search to perform",
         TR::Options::set32BitSignedNumeric, offsetof(OMR::Options,_lastSearchCount), 0, "F%d"},
+   {"skipIlgenOpts", "O\tDo not run the optimizer during ilgen", SET_OPTION_BIT(TR_DisableIlgenOpts), "F"},
```

Comment:
Please change the option name text here as well (and insert the new name alphabetically).